rust/Earthfile

```diff
@@ -109,7 +109,38 @@ COPY_OUTPUT:
             cd ..;  \
         fi;
     RUN mkdir -p target; \
-        mv $TMP_FOLDER/* target 2>/dev/null || echo "no files found within ./target matching the provided output regexp" ;
+        mv $TMP_FOLDER/* target 2>/dev/null || echo "no files found within ./target matching the provided output regexp";
+
+# CROSS runs the [cross](https://github.com/cross-rs/cross) command "cross $args --target $target".
+# Notice that in order to run this function, +INIT must be called first.
+# Arguments:
+#   - target: Cross [target](https://github.com/cross-rs/cross?tab=readme-ov-file#supported-targets). Required.
+#   - args: Cross subcommand and its arguments. By default: "build --release"
+#   - output: Regex matching output artifacts files to be copied to ./target folder in the caller filesystem (image layers). By default: "$target/release/[^\./]+"
+# Example:
+#   DO rust+SET_CACHE_MOUNTS_ENV
+#   DO rust+CROSS --target aarch64-unknown-linux-gnu
+CROSS:
+    FUNCTION
+    ARG --required target
+    ARG args="build --release"
+    ARG output="$target/release/[^\./]+"
+    ARG version=0.2.5
+    DO +SET_CACHE_MOUNTS_ENV --target_cache_suffix="$target"
+    RUN echo "$EARTHLY_RUST_TARGET_CACHE"
```

Comment:
is this intentional?

Reply:
nop, thanks!

Reply:
maybe it's a dumb question, but should there also be an `ARG EARTHLY_RUST_TARGET_CACHE` somewhere? I can't seem to find one while grepping.

Reply:
nevermind, it's an ENV:

```
rust/Earthfile:    ENV EARTHLY_RUST_TARGET_CACHE="type=cache,mode=0777,id=$EARTHLY_CACHE_PREFIX#target#$EARTHLY_TARGET_NAME#$TARGETPLATFORM#$target_cache_suffix,sharing=locked,target=target"
```

:)